package.json

```
@@ -3,7 +3,7 @@
   "private": true,
   "dependencies": {
```

```suggestion
  "devDependencies": {
```
A little thing. I known that It is not the PR to made this change. We have this suggestion in [doppler-email-address-validator](https://github.com/MakingSense/doppler-email-address-validator/blob/main/package.json#L5).

---

I think that I can add this change to this PR, thanks!

---

Fixed in #65